spec/spec_helper_acceptance.rb

```diff
@@ -1,22 +1,7 @@
-require 'beaker-rspec'
-require 'pry'
+require 'voxpupuli/acceptance/spec_helper_acceptance'
 
-hosts.each do |host|
-  # Install Puppet
-  on host, install_puppet
-end
-
-RSpec.configure do |c|
-  module_root = File.expand_path(File.join(File.dirname(__FILE__), '..'))
-
-  c.formatter = :documentation
-
-  # Configure all nodes in nodeset
-  c.before :suite do
-    # Install module
-    puppet_module_install(source: module_root, module_name: 'monit')
-    hosts.each do |host|
-      on host, puppet('module', 'install', 'puppetlabs-stdlib'), acceptable_exit_codes: [0, 1]
-    end
-  end
+configure_beaker do |host|
+  install_local_module_on(host)
+  install_puppet_module_via_pmt_on(host, 'puppetlabs-stdlib')
```

Comment:
I think `configure_beaker` should install the module and its dependencies by default
```suggestion
```

Reply:
is that the case by now? :D i nicked this from some other of our modules thats properly maintained ^^ i can certainly try to leave that out